public/cli/lib/commands/logger.ts

```diff
@@ -69,7 +69,14 @@ export async function prepareLogger(numLogsToKeep = 10, pathToLogDir = ''): Prom
               info: 'cyan',
             },
           }),
-          winston.format.simple()
+          winston.format.printf((info) => {
+            const {level, message, service, ...meta} = info;
+            if (Object.keys(meta).length){
```

Comment:
Its a good idea to be more explicit with checks. You don't just want length to exist and non zero but actually a positive length, i.e. `if (Object.keys(meta).length > 0) {`

Reply:
Done.